src/H5FDmirror.h

```diff
@@ -25,46 +25,54 @@
 /** Identifier for the mirror VFD */
 #define H5FD_MIRROR_VALUE H5_VFD_MIRROR
 
-#ifdef __cplusplus
-extern "C" {
-#endif
+/** Magic number to identify the H5FD_mirror_fapl_t struct */
+#define H5FD_MIRROR_FAPL_MAGIC 0xF8DD514C
 
-/* ============================================================================
- * Mirror VFD use and operation.
- * ============================================================================
+/**
+ * The version number of the H5FD_mirror_fapl_t configuration
+ * structure for the #H5FD_MIRROR driver
  */
+#define H5FD_MIRROR_CURR_FAPL_T_VERSION 1
 
-/* ---------------------------------------------------------------------------
- * Structure:   H5FD_mirror_fapl_t
+/** Max size of the remote_ip array in H5FD_mirror_fapl_t */
+#define H5FD_MIRROR_MAX_IP_LEN 32
```

Comment:
https://stackoverflow.com/questions/1076714/max-length-for-client-ip-address

Reply:
I'll update that in a separate PR